src/wingdi.rs

```diff
@@ -804,3 +840,259 @@ pub const FW_REGULAR: ::c_int = FW_NORMAL;
 pub const FW_DEMIBOLD: ::c_int = FW_SEMIBOLD;
 pub const FW_ULTRABOLD: ::c_int = FW_EXTRABOLD;
 pub const FW_BLACK: ::c_int = FW_HEAVY;
+
+pub type COLOR16 = ::c_ushort;
+
+#[repr(C)] #[derive(Clone, Copy, Debug, Default)]
```

Comment:
Please no `Default`. It is easy enough to use `mem::zeroed` and I don't want compile times impacted by the extra codegen.